man/systemd.resource-control.xml

```diff
@@ -1545,6 +1545,24 @@ DeviceAllow=/dev/loop-control
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>ManagedOOMMemoryPressureDurationSec=</varname></term>
+
+        <listitem>
+          <para>Overrides the default memory pressure duration set by
+          <citerefentry><refentrytitle>oomd.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry> for
+          this unit (cgroup). The specified value expects a time unit such as <literal>ms</literal> or
+          <literal>μs</literal>, see
+          <citerefentry><refentrytitle>systemd.time</refentrytitle><manvolnum>7</manvolnum></citerefentry>
+          for details on the permitted syntax.  This property is ignored unless
+          <varname>ManagedOOMMemoryPressure=</varname><option>kill</option>. Must be set to 0, or at least 1
+          second. Defaults to 0, which means to use the default set by
+          <citerefentry><refentrytitle>oomd.conf</refentrytitle><manvolnum>5</manvolnum></citerefentry>.</para>
```

Comment:
btw, what#s the rationale for using 0 as special value indicating "unset"? we typically use UINT64_MAX internally for that, and expect people assign an empty string. We should really do that here too.

Reply:
Rationale was ManagedOOMMemoryPressureLimit uses 0% as default: https://www.freedesktop.org/software/systemd/man/latest/systemd.resource-control.html#ManagedOOMMemoryPressureLimit=

So wanted to be consistent among OOMD settings in unit files

But to be honest, I agree unset is probably cleaner and more sensible to the user. Will switch to USEC_INFINITY!